test/index.test.js

```diff
@@ -28,9 +28,9 @@ it('Should replace strings in comments and styles.', () => {
 });
 
 it('Should throw a TypeError if invalid pattern is supplied.', () => {
-    return run('basic', { data: pkg, pattern: '' }).catch(e =>
-        expect(e).toBeInstanceOf(TypeError)
-    )
+    expect(() => {
+        return run('basic', { data: pkg, pattern: null })
```

Comment:
Changed `''` by `null` given that the original code tests for `typeof options.pattern === 'string'` I'm unsure how this worked before since an empty string is actually a string after all